website/learn/unique-semantics.md

```diff
@@ -323,7 +323,7 @@ representation.
 
 If any error happens during the first two phases, Elvish rejects the chunk
 without executing any of it. For instance, in Elvish unclosed parenthesis is a
-syntax error (error in the first parsing phase). The following code, when
+syntax error during the first, parsing, phase. The following code, when
```

Comment:
The commas indicate that the two words describe the same thing and is therefore correct grammar. However, I would argue that it would be better to simply say `... during the parsing phase.` Stating that is also the first phase is redundant. Especially since the three execution phases are defined in the previous paragraph. So unless there is a strong objection I'll simplify it.